capstone-frontend/src/components/Browse.jsx

```diff
@@ -14,10 +14,16 @@ export class Browse extends Component {
 
   getSearchedBooks = (searchQuery) => {
     fetch(`/api/search?searchTerm=${searchQuery}`)
-      .then(response => response.json())
+      .then(response => {
```

Comment:
could combine this into a one-line ternary if you'd like.

ie `.then(response => response.status === 200 ? response.json() : [])`

Reply:
Great idea! I didn't think of using a ternary here but it makes it much cleaner. I'll also replace it in the other 2 files